src/io.c

```diff
@@ -360,7 +360,7 @@ UInt OpenInput(TypInputFile * input, const Char * filename)
 #endif
 
     /* try to open the input file                                          */
-    file = SyFopen( filename, "r" );
+    file = SyFopen(filename, "r", FALSE);
```

Comment:
Shouldn't transparent decompression upon reading always be enabled? Also elsewhere... that might explain the test failures?
```suggestion
    file = SyFopen(filename, "r", TRUE);
```

Reply:
Yep, cleaning up